src/frontends/onnx/frontend/src/utils/pooling_factory.cpp

```diff
@@ -55,14 +55,15 @@ PoolingFactory::PoolingFactory(const Node& node)
 
 ov::OutputVector PoolingFactory::make_avg_pool() const {
     const bool count_include_pad = m_onnx_node.get_attribute_value<std::int64_t>("count_include_pad", 0);
-    return {std::make_shared<v1::AvgPool>(m_inputs.at(0),
-                                          m_strides,
-                                          m_padding_below,
-                                          m_padding_above,
-                                          m_kernel_shape,
-                                          !count_include_pad,
-                                          m_rounding_type,
-                                          m_auto_pad)};
+    return {std::make_shared<v16::AvgPool>(m_inputs.at(0),
+                                           m_strides,
+                                           m_dilations,
+                                           m_padding_below,
+                                           m_padding_above,
+                                           m_kernel_shape,
+                                           !count_include_pad,
+                                           m_rounding_type,
+                                           m_auto_pad)};
```

Comment:
To avoid regressions, I would suggest to produce the new version only if required (if dilations provided and not default, filled with ones). The `dilation` attribute is a known value at conversion step. This part can be unified to produce a single version, as proposed, when supported by all plugins, and compatibility with transformations ensured.